### DISCUSSION TITLE
Any good GUI libraries

### DISCUSSION THREAD

**Discussion by notmithun** [CosmosOS/Cosmos#3107] [General]
I'm not expecting anything modern, but are there any good GUI libraries you might use? I like the Cosmos's in-built one, but I'm just searching for alternatives.

**Comment by ghost** [comment]
there is one called PrismAPI

**Reply by notmithun** [reply]
Thanks, I'll check that out

**Reply by 9xbt** [reply]
I have a PrismAPI fork called GrapeGL which is PrismAPI stripped down to only have the graphics part and with bug fixes

**Reply by 9xbt** [reply]
> there is one called PrismAPI

That's dead you know, GrapeGL is dead too but PrismAPI has been dead for a while

**Reply by notmithun** [reply]
> PrismAPI fork called GrapeGL

In GrapeGL, it is mentioned that 

> This branch of GrapeGL is unstable and might have a few issues & WIP features

**Reply by notmithun** [reply]
sorry for the late reply

**Reply by 9xbt** [reply]
> In GrapeGL, it is mentioned that
> > This branch of GrapeGL is unstable and might have a few issues & WIP features

That's just me merging branches wrong

**Reply by notmithun** [reply]
oh ok lol

**Comment by ghost** [comment]
Here's A Window System If You Don't Have One:
https://github.com/WyattSteinmann/JXWS-Cosmos-Window-System
